contracts/util/TellerCommon.sol

```diff
@@ -56,7 +90,16 @@ library TellerCommon {
         uint256 requestTime;
     }
 
-    // Borrower response object to take out a loan
+    /**
+        @notice Borrower response object to take out a loan
+        @param signer The waller address of the signer validating the interest request of the lender
+        @param consensusAddress The address of the Teller loan consensus contract to which the request should be sent
+        @param responseTime The timestamp at which the response was sent
+        @param interestRate The signed interest rate generated by the signer's CRA
```

Comment:
Whats a CRA?

Reply:
Credit Risk Algorithm (CRA) - made the change